ast/visit_test.go

```diff
@@ -37,264 +38,10 @@ p { false } else { false } else { true }
 fn([x, y]) = z { json.unmarshal(x, z); z > y }
 `)
 	vis := &testVis{}
-
 	NewGenericVisitor(vis.Visit).Walk(rule)
 
-	/*
```

Comment:
Wow 😅

Reply:
It didn't really give me much, so I've removed it 😅